### PR TITLE
adding the endpoints for ads and medical service

### PR DESCRIPTION
**Pull request by Alberthor47**

### CODE REVIEW

File: routes/products.js [hunk 1]
```diff
@@ -11,9 +12,9 @@ let db = admin.firestore();
 
 function productsApi (app){
   const router = express.Router();
-  app.use('/api', router);
```
Comment (WilliamVelazquez) [inline routes/products.js:14]: You are adding `api` to all of the endpoints except for the Ads and Medical ones.
You should keep the prefix on all the routes.

File: routes/products.js [hunk 2]
```diff
@@ -119,6 +120,43 @@ function productsApi (app){
       res.status(500).json({"error":"deleting"});
     })
   })
+
+  router.get('/ads', (req, res, next) => {
```
Comment (WilliamVelazquez) [inline routes/products.js:124]: We need the Prefix Path `api` here!

File: routes/products.js [hunk 3]
```diff
@@ -119,6 +120,43 @@ function productsApi (app){
...
+      })
+      res.status(200).json(documentsId);
+    }) 
+    .catch((err) => {
+      console.log('Error getting documents in collection:', err);
+      res.status(500).json({"error":"getting documents in collection"});
+    })
+  })
+
+  router.get('/medic', (req, res, next) => {
```
Comment (WilliamVelazquez) [inline routes/products.js:142]: We need the Prefix Path `api` here!

**Comment by Alberthor47** [comment]
Done, all paths have the /api prefix and there were divided in two base structures, services and products

**Pull request merged**